packages/change-codemod/transforms/__testfixtures__/tap-to-then-with-omit-return.output.js

```diff
@@ -0,0 +1,14 @@
+someOperation.then((result) => {
+  return 'new-result';
+}).then((result) => {
+  // do something
+}).tap(inlineIdentifier)
+  .then((result) => inlineBlock())
+  .then(function(result) {
+    // second to last in chain
+    callFunction();
+  }).then(function() {
+  // last in chain
+  callFunction();
+  return result;
```

Comment:
If omitReturn = true, why is the last one in the chain returning `result`? Also, wouldn't this throw since return is not declared in that scope?

Reply:
Yeah that seems backward.  I had a hard time figuring out if this was a mistake or as intended.  I think the logic was backwards and the return should only be omitted if it's the last in the chain.  So I've updated the logic.